apps/wallet/src/ui/app/pages/accounts/ExportPublicKey.tsx

```diff
@@ -18,6 +20,15 @@ export function ExportPublicKeyPage() {
             if (!account) {
                 return null;
             }
+
+            if (account.type === AccountType.LedgerDerived) {
+                if (!account.publicKey) {
+                    throw new Error('Public key not available for this Ledger account');
```

Comment:
Don't know if the wallet stores the publicKey when it's created and this error handling is just to make TS happy, but if it's not stored, it could be requested from the ledger